test/AutoDiff/control_flow.swift

```diff
@@ -1,79 +1,138 @@
-// RUN: %target-swift-frontend -emit-sil -verify -Xllvm -differentiation-enable-control-flow %s | %FileCheck %s -check-prefix=CHECK-DATA-STRUCTURES
-// RUN: %target-swift-frontend -emit-sil -verify -Xllvm -differentiation-enable-control-flow %s | %FileCheck %s -check-prefix=CHECK-SIL
+// RUN: %target-run-simple-swift-control-flow-differentiation
+// REQUIRES: executable_test
 
-// Test cond. A simple if-diamond.
+import StdlibUnittest
 
-@differentiable
-@_silgen_name("cond")
-func cond(_ x: Float) -> Float {
-  if x > 0 {
+var ControlFlowTests = TestSuite("ControlFlow")
```

Comment:
`guard` tests should be added.

Reply:
Added in https://github.com/apple/swift/pull/25057/commits/bde639fd10de0912da43c49dfcf7417d279da4ff.